src/index.js

```diff
@@ -134,7 +135,8 @@ class Griddle extends Component {
 
     this.store = createStore(
       reducers,
-      initialState
+      initialState,
+      applyMiddleware(...plugins.map(p => p.reduxMiddleware), ...reduxMiddleware) 
     );
```

Comment:
Yes, correct. Flatten the plugins `reduxMiddleware` then remove and undefined values from that (for example if a plugin doesn't define `reduxMiddleware`).